app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -388,6 +388,16 @@ def traces_helpers(search:)
         ::Katello::HostTracer.helpers_for(traces)
       end
 
+      def package_names_for_job_template(feature:, search:)
+        features = ['katello_package_install_by_search']
```

Comment:
Maybe better not tight the feature name into the method which cares only if it is to remove, install or update the packages. So in case the feature name got changed later, this method won't be affected if the first parameter is based on feature type (like remove/install/update) instead of feature name.
But it is up to you.

Reply:
Sure, but I don't think I've ever seen a REX feature name actually change. I'd lean towards the side of trying not to prematurely optimize. But tbh I don't feel super strongly either way.

On the other hand, 'install' and 'remove' are a lot easier to type and read than 'katello_package_install_by_search' and 'katello_package_remove_by_search'. So I'll change it for that reason :)

Reply:
Updated